tests/pytests/unit/state/test_state_compiler.py

```diff
@@ -855,128 +862,6 @@ def test_call_chunk_sub_state_run(minion_opts):
             assert sub_state["__sls__"] == "external"
 
 
-def test_aggregate_requisites(minion_opts):
```

Comment:
Is there a reason why this test is removed?

Reply:
If I am remembering correctly (since I removed this half a year ago), because the test is not correct and is not how the ordering and mod aggregate works. If I am remembering correctly the requisites of the states stay the same and the network is just updated to determine the requisite ordering.